src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ItemTypeToolStripMenuItemTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Drawing;
+using System.Drawing.Design;
+
+namespace System.Windows.Forms.Design.Tests
+{
```

Comment:
Please use file scoped namespaces. The ending curly brace will need to be removed as well.
```suggestion
namespace System.Windows.Forms.Design.Tests;
```